### PR TITLE
Old apple clone

### PR DESCRIPTION
**Pull request by Davidosky007**
> #The Old Apple Website Clone
> 
> In this project we:
> 
> *we made use of background images
*we made use of Gradients
*added a favicon for the page title

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# This Pull Request follows observations made in PR #2. 

### NOTES: Please use always the same PR that you submit initially. It keeps track of history and reflects the observations and decisions made through the elaboration of the project.

# Status: Changes Required :recycle:

## :clap: :clap: Great work team!, you have solved some of all the previous required changes!

Still, there are some issues to be addressed. Kindly work on them before submitting them for a re-review.

## Required Changes
### [Mandatory]

- [ ] Website still is not displaying correctly
- [ ] Navbar links are difficult to perceive, we should increase the ```font-weight```.
image

- [ ]  Navbar should be inside the `<header>` tag. We should move it outside the `<main>` tag. See inline comments.

- [ ]  All images should have an alternate text. We should fill the `alt=" "` attribute, never leave it blank. I've seen you have fill it up with ```img``` text. We should improve and make a meaningful and descriptive alternative text.
**Actual**: ```alt="img"```
**Improved** :+1: : ```alt="Iphone 5S"```

### [OPTIONAL] 
- [ ]  The project should follow this structure:
 ```
/
   css/
     style.css
   img/
     ... some images...
   js/
     ... some js scripts...
   index.html
 ```
## [Previous Required Changes PR #2] -----------------------
> # Status: Changes Required 
> ##   Hello Team!! Great effort on the project!!
>  Great that you use media queries even if they were not required.
> 
> ## Required Changes
> ### [Mandatory] 
> * [x]  MIT license link takes to a Linkedin profile. It should take to the actual license.
- [ ]  Website is not displaying properly
>   image
- [ ]  Navbar link elements are not visible enough.
>   image
- [ ]  Navbar should be inside the `<header>` tag. We should move it outside the `<main>` tag.
> * [x]  Footer should be outside the `<main>` tag.
- [ ]  All images should have an alternate text. We should fill the `alt=" "` attribute, never leave it blank.
> 
> ### [OPTIONAL] 
> * [ ]  The project should follow this structure:
> 
> ```
> /
>   css/
>     style.css
>   img/
>     ... some images...
>   js/
>     ... some js scripts...
>   index.html
> ```
> 
> ### [Comments] 
> * [ ]  After the changes have been made do nor forget to update the live preview link in the README
> 
## =====================================

## Keep up the great work  and Happy Coding  !!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            <li><a href="#">Support</a></li>
+            <li>
+              <form action="#" class="main-search">
+                <input type="search" name="search" />
+                <i class="fas fa-search fa-sm"></i>
+              </form>
+            </li>
+
+          </ul>
+        </nav>
```
Comment (jubaan) [inline index.html]: - [ ] Navbar should be wrapped around ```<header>``` HTML tag, and outside the ```<main>``` container. Example:
```
...
  <header>
    <nav>
      ... nav elements ...
    </nav>
  </header>

  <main>
    ... main content ...
  </main>
...
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      </div>
+      <div>
+        <a href="#" class="sub-image"><img src="images/promo_iphone5c.jpg" alt="img" /></a>
+      </div>
+      <div>
+        <a href="#" class="sub-image"><img src="images/promo_30_years.jpg" alt="img" /></a>
+      </div>
+      <div>
+        <a href="#" class="sub-image"><img src="images/promo_supplier_responsibility.jpg" alt="img" /></a>
+      </div>
```
Comment (jubaan) [inline index.html]: - [ ] Give ```alt``` attribute descriptive alternative text as in the example given.

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Great job workking on the required changes! 
## Congratulations :clap: :clap: :clap:

## See you at the top :rocket: !!!

**Comment by Davidosky007** [comment]
Thanks

**Pull request merged**